src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+.img-responsive {

```

Comment:
define max_width class in reset.scss and add max-width:100% property and width:auto so we have both variations for the image with fluid width and max width.